src/components/Modal/Modal.Body.jsx

```diff
@@ -57,7 +57,7 @@ class ModalBody extends React.PureComponent {
 
     const componentClassName = classNames(
       'c-ModalBody',
-      v2 && 'is-v2',
```

Comment:
this is fine since you are saying it will only apply to hsds v3... i just want to note that there are two instances of style overrides in hsapp targeting the '.is-v2' class, so if you end up merging this into v2 it will cause issues.

Reply:
I think alert is not even used anywhere right now (maybe in setup-guide, but it's in the migration process), hence why I was thinking of keeping it to v3 only